### ISSUE TITLE
Timesketch output module generates a new entry in the db for search indexes

### ISSUE THREAD

**Issue by kiddinn** [log2timeline/plaso#3097]
**Description of problem:**

The timesketch output module calls:
```
    with self._timesketch.app_context():
      search_index = timesketch_sketch.SearchIndex.get_or_create(
          name=self._timeline_name, description=self._timeline_name, user=user,
          index_name=self._index_name)
```

The problem here is that the Timesketch server already creates the search index and it is not needed for the psort output module to do the same. The other thing is that Timesketch creates a search index with a particular username, whereas the plaso module does not (in the case of an uploaded plaso file where a celery worker ingests it). That means that the search query done in plaso does not return any record (since they pass `user=None` instead of `user=<valid user object>` like the Timesketch uploader does).

That means that there are two entries for each search index created in the database. One that Timesketch generates, and has proper ACLs and is tracked and updated via the celery workers, and one that the psort output module creates in the end, that stays in the `processing` state and has no ACLs, meaning that search index appears in a list for all authenticated users in the system.

This needs to be removed from the TS output module.

**Plaso version:**

HEAD

**Comment by joachimmetz** [comment]
Related https://github.com/log2timeline/plaso/issues/2560

**Comment by joachimmetz** [comment]
@kiddinn @berggren let's sync and come up with a plan to determine how to move forward with the timesketch and elastic output module

**Comment by joachimmetz** [comment]
As discussed @kiddinn will work on a new Timesketch output module that uses the Timesketch import API and we'll remove the old Timesketch output module once the new one is ready.

**Comment by joachimmetz** [comment]
Plan removal of timesketch output module

**Comment by kiddinn** [comment]
With #3470, TS will now move over to use the newly added `elastic_ts` module instead of the timesketch one. This should fix the issue.
